### PR TITLE
Don't report integration tests for non prod envs

### PR DESCRIPTION
**Pull request by emord**
@benrudolph I think these are the right set of checks. What's the best way to test changes like this?

buddy @czue

### CODE REVIEW

File: corehq/apps/hqadmin/management/commands/record_deploy_success.py [hunk 1]
```diff
@@ -71,21 +71,24 @@ def handle(self, *args, **options):
...
-            "Monitor the {{dashboard_link}}. "
-            "Check the integration {{integration_tests_link}}. "
-            "Find the diff {{diff_link}}".format(
-                options['environment'],
-                options['user'],
-                minutes or '?',
-            )
         )
 
+        if options['environment'] = 'production':
```
Comment (dimagimon) [inline corehq/apps/hqadmin/management/commands/record_deploy_success.py]: E901 SyntaxError: invalid syntax

File: corehq/apps/hqadmin/management/commands/record_deploy_success.py [hunk 2]
```diff
@@ -71,21 +71,24 @@ def handle(self, *args, **options):
...
+
         if settings.MOBILE_INTEGRATION_TEST_TOKEN:
+            deploy_notification_text += "Check the integration {{integration_tests_link}}. "
             requests.get(
                 'https://jenkins.dimagi.com/job/integration-tests/build',
                 params={'token': settings.MOBILE_INTEGRATION_TEST_TOKEN},
             )
 
+        deploy_notification_text += "Find the diff {{diff_link}}".format(
+                options['environment'],
```
Comment (dimagimon) [inline corehq/apps/hqadmin/management/commands/record_deploy_success.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/hqadmin/management/commands/record_deploy_success.py [hunk 3]
```diff
@@ -71,21 +71,24 @@ def handle(self, *args, **options):
...
+                options['user'],
+                minutes or '?',
+            )
```
Comment (dimagimon) [inline corehq/apps/hqadmin/management/commands/record_deploy_success.py]: E121 continuation line under-indented for hanging indent

**Comment by benrudolph** [comment]
yay! thanks, been wanting to do this for a while but have been lazy

**Comment by benrudolph** [comment]
killing build and merging since there are no tests

**Comment by czue** [comment]
nice

**Pull request merged**